File: PCMC/Entities/School.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace PCMC.Entities
{
    public class School
    {
        public int ID { get; set; }
        public string Name { get; set; }

        public Instructor Instructor { get; set; }
        public ICollection<Team> Teams { get; set; }
    }
}

File: PointSaaS/Client/Common/POCO/School.ts
class School {

    public constructor(id: number, name: string) {
        this.ID = id;
        this.Name = name;
    }

    public ID: number;
    public Name: string;
    // Won't really be used...
    public Instructor: Instructor;
    public Teams: Array<Team>;

}
